src/main/java/com/example/demo/service/ProductServiceimpl.java

```diff
@@ -1,81 +1,77 @@
- package com.example.demo.service;
+package com.example.demo.service;
 
 import java.util.Collection;
-import java.util.List;
 import java.util.Optional;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
-import com.example.demo.dao.Customerdao;
 import com.example.demo.dao.Productdao;
-import com.example.demo.model.Customer;
 import com.example.demo.model.Product;
 
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+@Data
+@Slf4j
 @Service
 public class ProductServiceimpl implements ProductsService {
 
 	@Autowired
-	Productdao product;
-	
-	@Autowired
-	Customerdao customer;
-	
-	public void createProduct(Product product2 , Customer customer1) {
-		// TODO Auto-generated method stub	
-	    customer.save(customer1);
-		product.save(product2);
-		
+	Productdao productdao;
```

Comment:
Class name should be PascalCase and variable name should be camelCase

Reply:
Done.